functional-tests/tests/tests.js

```diff
@@ -25,68 +23,60 @@ module.exports = {
 		browser.end();
 	},
 	'Check Tag in search box for URL with tag filter': function (browser) {
-		browser.url("http://bagcupid.com/app/tags/3")
-		browser.pause(7000);
-		browser.assert.visible(".select2-container--default .select2-selection--multiple .select2-selection__choice");
+	    browser.url("http://bagcupid.com/app/tags/3");
+	    browser.waitForElementVisible(".select2-container--default .select2-selection--multiple .select2-selection__choice", 7000);
```

Comment:
These two lines have mixed tabs/spaces.  Most editors have a way of alerting you when this happens, which do you use?